lib/src/collect.dart

```diff
@@ -12,6 +12,9 @@ const _retryInterval = const Duration(milliseconds: 200);
 Future<Map<String, dynamic>> collect(
```

Comment:
Would it be worth documenting this and saying that `serviceUri` needs to be non-null?

Reply:
Definitely! Done.